Automattic/newspack-newsletters
# Add preview modal

### All Submissions:






### Changes proposed in this Pull Request:



Adds a preview modal instead of preview link.

### How to test the changes in this Pull Request:

1. Open a newsletter or create a new one
2. In the sidebar, click "Preview" button
3. Observe a modal with preview. In top left, icons allow to change responsive presets.

### Other information:
